data/wof_snapshot.py

```diff
@@ -117,13 +119,12 @@ def __exit__(self, type, value, traceback):
         item = matching[0]
 
         version = item['last_updated']
-        count = item['count']
         download_size = item['size_compressed']
 
-        print "Downloading %r with %d entries" % (placetype, count)
+        print "Downloading %r" % (placetype)
         with tmpdownload(WOF_BUNDLE_PREFIX + fname, download_size) as fname:
             print "Parsing WOF data"
-            reader.add_archive(fname, version, count)
+            reader.add_archive(fname, version, 1)
```

Comment:
Add a comment here saying why the count is always 1, please. Something like `20210820: geocode.earth inventory files don't offer count, so set this to 1 all the time`